static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -1,3 +1,5 @@
+$mobileMedium: 375px; // Breakpoint for our most common device sizes
```

Comment:
Well, indeed. I have a PR prepared for doing just that but it's potentially ruffling some feathers. Think we all need to discuss if it's necessary or not.